src/EventStore.Core/ClusterVNodeOptions.cs

```diff
@@ -687,11 +687,6 @@ public record InterfaceOptions {
 				}
 			}
 
-			[Description("Whether to enable external TCP communication"),
```

Comment:
Just making some notes of TCP related db settings and how this PR affects them:

| Old db setting | Replaced with env var |
|---|---|
| EnableExternalTcp | EVENTSTORE_UNSUPPORTED_TCP_API_ENABLED |
| ExtTcpPort / NodeTcpPort | EVENTSTORE_UNSUPPORTED_TCP_API_PORT |
| ExtHostAdvertiseAs / NodeHostAdvertiseAs | (kept) |
| ExtTcpPortAdvertiseAs / NodeTcpPortAdvertiseAs | (kept) |
| ExtIp / NodeIp | (kept) |
| ExtTcpHeartbeatTimeout / NodeHeartbeatTimeout | (kept) |
| ExtTcpHeartbeatInterval / NodeHeartbeatInterval | (kept) |

`EVENTSTORE_UNSUPPORTED_TCP_API_ADVERTISED_PORT` is a new environment variable but `ExtTcpPortAdvertiseAs` is still present

I think we need to write down something about why we're keeping some but not others. Personally I would git rid of everything that is used only for the ext tcp interface that we are removing

Reply:
Now the only params remaining are `ExtIp / NodeIp` and `ExtHostAdvertiseAs / NodeHostAdvertiseAs` because those are also used by the http interface. Others have their environment variable name equivalent.

Reply:
DisableExternalTcpTls | n/a (this was unused anyway)

hey @timothycoleman @YoEight, this is actually used